CMakeLists.txt

```diff
@@ -44,28 +43,22 @@ catkin_package(
     roscpp
     rospy
     message_generation
+    message_runtime
 )
 
 include_directories(
   ${catkin_INCLUDE_DIRS}
 )
+
 # Declare a C++ library
-set(SRCS
+set(ros_alarms_srcs
  src/${PROJECT_NAME}/broadcaster.cpp
  src/${PROJECT_NAME}/alarm_proxy.cpp
 )
-add_library(ros_alarms ${SRCS})
+add_library(${PROJECT_NAME} ${ros_alarms_srcs})
 target_include_directories(${PROJECT_NAME} PUBLIC include)
 target_compile_options(${PROJECT_NAME} PRIVATE -g -Wall -std=c++11)
-
-add_dependencies(ros_alarms ros_alarms_generate_messages_cpp)
-
-#if(CATKIN_ENABLE_TESTING)
-#  foreach(T
-#    test/test_ros_alarms.test)
-#     add_rostest(${T})
-#  endforeach()
-#endif()
+add_dependencies(ros_alarms ${PROJECT_NAME}_generate_messages_cpp ${catkin_EXPORTED_TARGETS})
```

Comment:
Should be ```${${PROJECT_NAME}_EXPORTED_TARGETS}``` as per [ros wiki](http://docs.ros.org/hydro/api/catkin/html/howto/format2/building_msgs.html)